cpu/esp32/stdio_usb_serial_jtag/usb_serial_jtag.c

```diff
@@ -128,4 +106,11 @@ void stdio_init(void)
     intr_cntrl_ll_set_int_level(CPU_INUM_SERIAL_JTAG, 1);
 #endif
 }
+
+static void _detach(void)
```

Comment:
It gets used by `stdio_close()` - which doesn't have a use currently, but I thought it could be used to e.g. provide a debug shell on startup and once that's closed disable stdio to save power.